frameworks/cassandra/universe/resource.json

```diff
@@ -10,7 +10,7 @@
       "cassandra-jre-tar-gz": "{{jre-url}}",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
-      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.14-bin-dcos.tar.gz",
```

Comment:
@gabrielhartmann did you build this using https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/modify-cassandra-distribution.sh?

Reply:
yes